x/evm/keeper/keeper_test.go

```diff
@@ -95,17 +93,17 @@ func (suite *KeeperTestSuite) TestGetAccountStorage() {
 	testCases := []struct {
 		name     string
 		malleate func() common.Address
+		expRes   []int
```

Comment:
I believe this is not used because you restored the old logic but keeping the new test struct.
```suggestion
```